test/conftest.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+import pytest
+
+
+@pytest.yield_fixture
```

Comment:
`yield_fixture` is deprecated, just `fixture` will work: https://docs.pytest.org/en/latest/yieldfixture.html